[2022-03-15 21:29:20,429] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: gcs_ingestion_dag_v02.extract_from_zip_task scheduled__2020-01-01T00:00:00+00:00 [queued]>
[2022-03-15 21:29:20,436] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: gcs_ingestion_dag_v02.extract_from_zip_task scheduled__2020-01-01T00:00:00+00:00 [queued]>
[2022-03-15 21:29:20,436] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 21:29:20,436] {taskinstance.py:1239} INFO - Starting attempt 12 of 12
[2022-03-15 21:29:20,436] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 21:29:20,444] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): extract_from_zip_task> on 2020-01-01 00:00:00+00:00
[2022-03-15 21:29:20,449] {standard_task_runner.py:52} INFO - Started process 8066 to run task
[2022-03-15 21:29:20,451] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'gcs_ingestion_dag_v02', 'extract_from_zip_task', 'scheduled__2020-01-01T00:00:00+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/gcs_ingestion_dag_v02.py', '--cfg-path', '/tmp/tmpnjf99_02', '--error-file', '/tmp/tmprgtykcqx']
[2022-03-15 21:29:20,452] {standard_task_runner.py:77} INFO - Job 75: Subtask extract_from_zip_task
[2022-03-15 21:29:20,487] {logging_mixin.py:109} INFO - Running <TaskInstance: gcs_ingestion_dag_v02.extract_from_zip_task scheduled__2020-01-01T00:00:00+00:00 [running]> on host 3570d6d4f2dc
[2022-03-15 21:29:20,511] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-03-15 21:29:20,523] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=gcs_ingestion_dag_v02
AIRFLOW_CTX_TASK_ID=extract_from_zip_task
AIRFLOW_CTX_EXECUTION_DATE=2020-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-01T00:00:00+00:00
[2022-03-15 21:29:20,524] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-03-15 21:29:20,524] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'jar xvf /opt/***/bandcamp-items-json.zip albums-json/albums-full-info-1.json']
[2022-03-15 21:29:20,533] {subprocess.py:85} INFO - Output:
[2022-03-15 21:29:20,535] {subprocess.py:89} INFO - bash: jar: command not found
[2022-03-15 21:29:20,535] {subprocess.py:93} INFO - Command exited with return code 127
[2022-03-15 21:29:20,545] {taskinstance.py:1700} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1329, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1455, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1511, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 188, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2022-03-15 21:29:20,551] {taskinstance.py:1277} INFO - Marking task as FAILED. dag_id=gcs_ingestion_dag_v02, task_id=extract_from_zip_task, execution_date=20200101T000000, start_date=20220315T212920, end_date=20220315T212920
[2022-03-15 21:29:20,560] {standard_task_runner.py:92} ERROR - Failed to execute job 75 for task extract_from_zip_task
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/task/task_runner/standard_task_runner.py", line 85, in _start_by_fork
    args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/cli_parser.py", line 48, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/cli.py", line 92, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 298, in task_run
    _run_task_by_selected_method(args, dag, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 107, in _run_task_by_selected_method
    _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/cli/commands/task_command.py", line 184, in _run_raw_task
    error_file=args.error_file,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 70, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1329, in _run_raw_task
    self._execute_task_with_callbacks(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1455, in _execute_task_with_callbacks
    result = self._execute_task(context, self.task)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 1511, in _execute_task
    result = execute_callable(context=context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 188, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2022-03-15 21:29:20,583] {local_task_job.py:154} INFO - Task exited with return code 1
[2022-03-15 21:29:20,612] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
